tests/ColorSensorTest.cpp

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file ColorSensorTest.cpp
+ * @brief ColorSensorクラスの値取得をテストする
+ * @author HaruArima08
+ */
+
+#include "ColorSensor.h"
+#include <gtest/gtest.h>
+
+// pup_color_rgb_tの比較用関数
+bool eqRgb(pup_color_rgb_t rgb1, pup_color_rgb_t rgb2)
+{
+  return rgb1.r == rgb2.r && rgb1.g == rgb2.g && rgb1.b == rgb2.b;
+}
+
+// pup_color_hsv_tの比較用関数
+bool eqHsv(pup_color_hsv_t hsv1, pup_color_hsv_t hsv2)
+{
+  return hsv1.h == hsv2.h && hsv1.s == hsv2.s && hsv1.v == hsv2.v;
+}
+
+namespace etrobocon2025_test {
+
+  // RGB値取得テスト
+  TEST(RgbColorTest, getRGB)
```

Comment:
気が回らなかったけど、テスト名はアッパーキャメルでお願いします。一応コーディング規約そうなってます🙏他のも。信じてapproveはそのままにしときます。
```suggestion
  TEST(RgbColorTest, GetRGB)
```

Reply:
衝撃の事実、これは、、、去年の人たちが悪いね！

大変申し訳ございませんでした。

Reply:
実は僕がやりました。公式やらどこ見てもアッパキャメルだったので、ここを区切りにしたかったです。個人的にはアッパーの方が大文字でぱっと見で読みやすいのもある。